x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('RuleActionsOverflow', () => {
     jest.clearAllMocks();
   });
   afterAll(() => {
-    jest.resetAllMocks();
```

Comment:
@michaelolo24 @kqualters-elastic  There are large number of change like this becuase `resetAllMocks` were resetting saved search mocks to `jest.fn()` which was causing multitude of errors. 

`clearAllMocks` is only needed here.